### PR TITLE
Jan sweep

### PR DESCRIPTION
**Pull request by rrigdon**
I checked every conda-build page. Most of the edits are grammatical/formatting in nature. The define-metadata, use-shared-libraries, and variants docs need careful review for accuracy.

### CODE REVIEW

File: docs/source/concepts/channels.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Identical channel and package name problem
 
 If the channel and package name are identical, it's possible to encounter a build problem if the short channel name is used.
 
-Let's say your anaconda username or an organization name is ``example``. And suppose you created a package ``example``, whose files' layout is similar to:
+Let's say your Anaconda username or an organization name is ``example``. And suppose you created a package ``example``, whose files' layout is similar to:
```
Comment (msarahan) [inline docs/source/concepts/channels.rst]:
```suggestion
Let's say your Anaconda.org username or an organization name is ``example``. And suppose you created a package ``example``, whose files' layout is similar to:
```

???

File: docs/source/resources/build-scripts.rst [hunk 2]
```diff
@@ -25,9 +25,9 @@ As of conda-build 2.1, you can also define multiple output packages. Each
 package has its own script or list of files to include. The rules for these
 outputs are documented at :ref:`package-outputs`. When any output is defined,
 this overrides the default behavior of bundling anything in ``$PREFIX``. So
-to output multiple packages from a single recipe, remove any installation steps
-from ``build.sh`` or ``bld.bat`` and do them instead in your install script(s)
-for each output.
+to out put multiple packages from a single recipe, remove any installation
```
Comment (msarahan) [inline docs/source/resources/build-scripts.rst]:
```suggestion
to output multiple packages from a single recipe, remove any installation
```

File: docs/source/resources/compiler-tools.rst [hunk 3]
```diff
@@ -211,16 +211,16 @@ This design is intended to make it easy for you to customize your own compiler
...
+remember that choosing the wrong flags can reduce security, reduce performance,
 and cause incompatibilities.
 
-With that warning in mind, let's look at good ways to customize clang.
+With that warning in mind, let's look at good ways to customize Clang.
 
-1. Download or fork the code from https://github.com/anacondarecipes/aggregate .
-   The clang package recipe is in the clang folder. The main material is in the
+1. Download or fork the code from https://github.com/anacondarecipes/aggregate.
+   The Clang package recipe is in the Clang folder. The main material is in the
```
Comment (msarahan) [inline docs/source/resources/compiler-tools.rst]:
```suggestion
   The Clang package recipe is in the ``clang`` folder. The main material is in the
```

folder is case sensitive and is lowercase

File: docs/source/resources/compiler-tools.rst [hunk 4]
```diff
@@ -275,7 +275,7 @@ With that warning in mind, let's look at good ways to customize clang.
        find . -name "*activate*.sh.bak" -exec rm "{}" \;
 
 4. With those changes to the activate scripts in place, it's time to move on to
-   installing things. Look back at the clang folder's ``meta.yaml``. Here's
+   installing things. Look back at the Clang folder's ``meta.yaml``. Here's
```
Comment (msarahan) [inline docs/source/resources/compiler-tools.rst]:
```suggestion
   installing things. Look back at the ``clang`` folder's ``meta.yaml``. Here's
```

File: docs/source/resources/compiler-tools.rst [hunk 5]
```diff
@@ -288,12 +288,12 @@ With that warning in mind, let's look at good ways to customize clang.
              - clang {{ version }}
 
    The script reference here is another place you might add customization.
-   You'll either change the contents of those install scripts, or change the
+   You'll either change the contents of those install scripts or change the
    scripts that those install scripts are installing.
 
-   Note that we make the package ``clang`` in the main material agree in version
+   Note that we make the package ``Clang`` in the main material agree in version
```
Comment (msarahan) [inline docs/source/resources/compiler-tools.rst]:
```suggestion
   Note that we make the package ``clang`` in the main material agree in version
```
Package names are always all lowercase

File: docs/source/resources/compiler-tools.rst [hunk 6]
```diff
@@ -288,12 +288,12 @@ With that warning in mind, let's look at good ways to customize clang.
...
    with our output version. This is implicitly the same as the top-level
-   recipe. The ``clang`` package sets no environment variables at all, so it
+   recipe. The ``Clang`` package sets no environment variables at all, so it
```
Comment (msarahan) [inline docs/source/resources/compiler-tools.rst]:
```suggestion
   recipe. The ``clang`` package sets no environment variables at all, so it
```

File: docs/source/resources/compiler-tools.rst [hunk 7]
```diff
@@ -317,7 +317,7 @@ With that warning in mind, let's look at good ways to customize clang.
    Activate scripts are named according to our package name so they won't
    conflict with other activate scripts.
 
-   The symlink for clang is a clang implementation detail that sets the host
+   The symlink for Clang is a Clang implementation detail that sets the host
```
Comment (msarahan) [inline docs/source/resources/compiler-tools.rst]: ambivalent on caps here.  I lean towards lowercase, but not strongly.

File: docs/source/resources/define-metadata.rst [hunk 8]
```diff
@@ -757,14 +757,14 @@ when people install your metapackage.  This is important, because if
 ``run_exports`` affected transitive dependencies, you would see many added
 dependencies to shared libraries where they are not actually direct
 dependencies. For example, Python uses bzip2, which can use ``run_exports`` to
-make sure that people use a compatible build of bzip2. If people list python as
-a build time dependency, bzip2 should only be imposed for python itself, and
+make sure that people use a compatible build of bzip2. If people list Python as
```
Comment (msarahan) [inline docs/source/resources/define-metadata.rst]:
```suggestion
make sure that people use a compatible build of bzip2. If people list python as
```
package name, lowercase

File: docs/source/resources/define-metadata.rst [hunk 9]
```diff
@@ -1201,7 +1202,7 @@ contents.
 Scripts that create or move files into the build prefix can be
 any kind of script. Known script types need only specify the
 script name. Currently the list of recognized extensions is
-py, bat, ps1 and sh.
+py, bat, ps1, and sh.
```
Comment (msarahan) [inline docs/source/resources/define-metadata.rst]:
```suggestion
py, bat, ps1 and sh.
```
take your oxford comma and get the hell out of here

**Comment by msarahan** [comment]
Seems fine to me.  Please rebase and fix the merge conflict.

File: docs/source/resources/compiler-tools.rst (hunk 7 above)
Comment (rrigdon) [reply docs/source/resources/compiler-tools.rst]: Their documentation capitalizes it.

**Pull request merged**